### PR TITLE
Refactor show method

### PR DESCRIPTION
**Pull request by techprojsforfun**
add refactorings to remove code smells

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -13,87 +88,14 @@ def show
...
-				 'calories' => 200,
-				 'readyInMinutes' => 45,
-				 'price' => 8	
-				 }
-			]
-		}
 
-		@all_recipes = Recipe.find_in_api(@calories, @budget, @time)
+	def show
+		set_vars_from_curr_user
```
Comment (ahz0001) [inline app/controllers/users_controller.rb:96]: moving all_recipes out of the show method makes this so much cleaner

Comment (ahz0001) [reply app/controllers/users_controller.rb:96]: I think over all the show method is a bit messy, and this is a step in the right direction. good job!

**Comment by cheolhojang** [comment]
good refactoring; nice move on not having too many variables for find in api

**Review by cheolhojang — COMMENTED** [review]
Nice move on not having too many variables on the find in api method

**Review by ahz0001 — APPROVED** [review]

**Pull request merged**